### PR TITLE
Feature/remove imagestream mangling

### PR DESCRIPTION
**Pull request by clemensutschig**
fix for #39

### CODE REVIEW

File: ocp-templates/Dockerfile [hunk 1]
```diff
@@ -1,29 +1,10 @@
...
-    tar zxvf /tmp/oc.tar.gz -C /tmp/ && \ 
-    mv /tmp/oc /usr/bin/ && \
-    rm -rf /tmp/oc.tar.gz  && \
-    oc completion bash > ~/oc-bash-completion.sh && \
-    echo source ~/oc-bash-completion.sh >> ~/.bashrc && \
-    oc version
+RUN oc version
 
 WORKDIR /data
 COPY scripts /data/
```
Comment (michaelsauter) [inline ocp-templates/Dockerfile]: Newline at end of files please :)

File: ocp-templates/scripts/import_ocp_project_metadata.sh [hunk 2]
```diff
@@ -463,35 +463,20 @@ do
...
 		ref_imagestreamOwningProject=$(echo $ref_imagestreamWithRegistry | cut -d '/' -f2)
 		ref_imagestreamName=$(echo $ref_imagestreamWithRegistry | cut -d '/' -f3)
 		
 		echo " --> checking for referenced project image - stream: $ref_imagestreamWithRegistry"
 		
-		# add the internal cname as well to the check
-		if  [[ "$ref_imagestreamWithRegistry" == *"$OD_OCP_DOCKER_REGISTRY_SOURCE_IP"* || "$ref_imagestreamWithRegistry" == *"$OD_OCP_DOCKER_REGISTRY_SOURCE_INTERNAL_HOST"* ]] && [ ! -z ${OD_OCP_SOURCE_TOKEN// } ] && [ ! "$ocp_proj_namespace_suffix" == "cd" ]; then 
-			echo "Replacing internal $curr_ocp_namespace image stream - with OD registry reference"
+		# dont do any import on shared images from shared-image namespace and from CD
+		if [ ! -z ${OD_OCP_SOURCE_TOKEN// } ] && [[ ! "$ocp_proj_namespace_suffix" == "cd" ]] && [[ $ref_imagestreamOwningProject != *shared-services* ]] && [[ $ref_imagestreamOwningProject != *cd* ]]; then
```
Comment (michaelsauter) [inline ocp-templates/scripts/import_ocp_project_metadata.sh]: Should we compare exactly? `*cd*` looks dangerous to me ...

File: ocp-templates/Dockerfile (hunk 1 above)
Comment (clemensutschig) [reply ocp-templates/Dockerfile]: done

File: ocp-templates/scripts/import_ocp_project_metadata.sh (hunk 2 above)
Comment (clemensutschig) [reply ocp-templates/scripts/import_ocp_project_metadata.sh]: agreed - changed to "cd"

**Pull request merged**